infretis/classes/formats/formatter.py

```diff
@@ -581,7 +582,7 @@ def open_file_write(self):
                             'Overwriting existing file "%s"', self.filename
                         )
             self.fileh = open(self.filename, self.file_mode)
-        except (OSError, IOError) as error:  # pragma: no cover
```

Comment:
Not sure if this correction is correct?

Reply:
They are the same since Python 3.3: https://docs.python.org/3/whatsnew/3.3.html#pep-3151